From 527537027b0c9134192663c9d3ec01d6cea6c5c2 Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Mon, 7 Nov 2022 14:32:59 -0800
Subject: [PATCH] Move AM_PATH_GPG_ERROR before AM_PATH_GPGME.

The former sets $GPGRT_CONFIG which the latter needs to find and use
gpgrt-config instead of gpgme-config.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index cacdf013..2e38c49f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -180,6 +180,7 @@ fi
 AC_MSG_CHECKING([whether to build with GPGME support])
 if test x"$mutt_enable_gpgme" = xyes; then
    AC_MSG_RESULT(yes)
+   AM_PATH_GPG_ERROR(1.33)
    AM_PATH_GPGME([$mutt_gpgme_version], AC_DEFINE(CRYPT_BACKEND_GPGME, 1,
 		 [Defined, if GPGME support is enabled]),
 		 [gpgme_found=no])
@@ -187,7 +188,6 @@ if test x"$mutt_enable_gpgme" = xyes; then
       AC_MSG_ERROR([*** GPGME not found or version is older than $mutt_gpgme_version ***])
    else
       MUTT_LIB_OBJECTS="$MUTT_LIB_OBJECTS crypt-gpgme.o crypt-mod-pgp-gpgme.o crypt-mod-smime-gpgme.o"
-      AM_PATH_GPG_ERROR(1.33)
    fi
 else
    AC_MSG_RESULT([no])
-- 
GitLab

